.travis.yml

```diff
@@ -47,7 +47,7 @@ before_script:
   - composer config -q github-oauth.github.com $GITHUB_OAUTH_TOKEN
   - if [ "$SYMFONY_VERSION" = "2.8.*@dev" ] || [ "$SYMFONY_VERSION" = "3.0.x-dev as 2.8" ]; then SYMFONY_DEPRECATIONS_HELPER=strict; fi;
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
-  - travis_wait composer update --prefer-dist --no-interaction $COMPOSER_FLAGS
+  - travis_wait 40 composer update --prefer-dist --no-interaction $COMPOSER_FLAGS
```

Comment:
Looks like initial composer update takes a large amount of time on travis.
The initial update for php 5.5, php 5.6 took around 40 minutes, after the inital cache they are running at 4-8 minutes.
Currently trying to work out another way of getting php 5.3 & 5.4 up and running (they time out at the 2 hour travis cap)